src/Microsoft.AspNetCore.Blazor.Browser.JS/src/GlobalExports.ts

```diff
@@ -12,7 +14,7 @@ if (typeof window !== 'undefined') {
 
     _internal: {
       attachRootComponentToElement,
-      renderBatch,
+      renderBatch: (browserRendererId: number, batchAddress: Pointer) => renderBatch(browserRendererId, new SharedMemoryRenderBatch(batchAddress)),
```

Comment:
Currently it's hard-coded here to use `SharedMemoryRenderBatch`, but later we'll have different startup logic depending on the hosting model.